README.md

```diff
@@ -2,7 +2,11 @@
 
 > an interactive explorer for single-cell transcriptomics data
 
-[![DOI](https://zenodo.org/badge/105615409.svg)](https://zenodo.org/badge/latestdoi/105615409)
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3338548.svg)](https://doi.org/10.5281/zenodo.3338548)
```

Comment:
Why the change here? This is a static DOI for the 0.11.0 release. The current badge will automatically update the badge and link as we cut new releases.

Reply:
Not sure on the reasoning but the provided badge on https://zenodo.org/record/3235021#.XO8extNKjBI has changed to my diff

Reply:
more context on how zenodo handles DOI versioning: http://help.zenodo.org/#versioning